ygot/struct_validation_map_test.go

```diff
@@ -1115,18 +1201,31 @@ func TestCopyStruct(t *testing.T) {
 			},
 		},
 	}, {
-		name: "unimplemented: struct map with overlapping contents",
+		name: "struct map with overlapping contents",
 		inSrc: &copyTest{
 			StructMap: map[copyMapKey]*copyTest{
 				{"fourpure"}: {StringField: String("session-ipa")},
 			},
 		},
 		inDst: &copyTest{
 			StructMap: map[copyMapKey]*copyTest{
-				{"fourpure"}: {StringField: String("oatmeal-stout")},
+				{"fourpure"}: {
+					Uint32Field:  Uint32(42),
+					Uint16Field:  Uint16(16),
+					Float64Field: Float64(42.42),
+				},
```

Comment:
Check case where inDst also has StringField. Actually it's unclear what happens in such a case, I'd assume error.

Reply:
Done - PTAL. This is an error, yes, since the same logic is implemented for any struct (overlapping fields that aren't maps are treated as errors).